include/psa/crypto_config.h

```diff
@@ -2101,35 +1995,6 @@
  */
 //#define MBEDTLS_CIPHER_NULL_CIPHER
 
-/**
- * \def MBEDTLS_CIPHER_PADDING_PKCS7
- *
- * MBEDTLS_CIPHER_PADDING_XXX: Uncomment or comment macros to add support for
- * specific padding modes in the cipher layer with cipher modes that support
- * padding (e.g. CBC)
- *
- * If you disable all padding modes, only full blocks can be used with CBC.
- *
- * Enable padding modes in the cipher layer.
- */
-#define MBEDTLS_CIPHER_PADDING_PKCS7
-#define MBEDTLS_CIPHER_PADDING_ONE_AND_ZEROS
-#define MBEDTLS_CIPHER_PADDING_ZEROS_AND_LEN
```

Comment:
`MBEDTLS_CIPHER_PADDING_ZEROS_AND_LEN` and `MBEDTLS_CIPHER_PADDING_ZEROS` have been removed but there is still code using them in drivers -> [cipher.h ](https://github.com/Mbed-TLS/TF-PSA-Crypto/blob/development/drivers/builtin/include/mbedtls/cipher.h#L196)
and [tests/suites/test_suite_cipher.function](https://github.com/Mbed-TLS/TF-PSA-Crypto/blob/development/tests/suites/test_suite_cipher.function#L176)

Is that intended to be cleaned up in a future pr?